browser/page_mapping.go

```diff
@@ -318,8 +318,12 @@ func mapPage(vu moduleVU, p *common.Page) mapping { //nolint:gocognit,cyclop
 				return nil, p.Uncheck(selector, opts) //nolint:wrapcheck
 			})
 		},
-		"url":          p.URL,
-		"viewportSize": p.ViewportSize,
+		"url": p.URL,
+		"viewportSize": func() *goja.Promise {
```

Comment:
This doesn't need to be a promise since:

1. It's not in Playwright
2. it's from memory.

Unless i missed something?

Reply:
Nice catch! Thanks 👍 Fixed here: [7608b48](https://github.com/grafana/xk6-browser/pull/1355/commits/7608b48cc090c346a36fde1b42819ea76a45d15e).